pkg/stores/proxy/unformatter_test.go

```diff
@@ -0,0 +1,110 @@
+package proxy
+
+import (
+	"testing"
+
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_unformat(t *testing.T) {
```

Comment:
Would it be better if this tested the exported method `Update` on the type instead?
`unformat` is just an implementation detail and might change, while we want to be certain that the public interface works as expected.

Reply:
In this case I don't find a lot of value in it. I would have to implement a mock store to hold the underlying `types.Store`, so it would be adding a lot of boilerplate to test the entire `.Update()` flow, when all that really matters is this one function.

Reply:
Makes sense. I just thought there might already be a simple mock store for testing...